cloudinstall/charms/quantum.py

```diff
@@ -77,11 +77,14 @@ def post_proc(self):
         utils.remote_cp(
             unit.machine_id,
             src=os.path.join(self.config.tmpl_path, "quantum-network.sh"),
-            dst="/tmp/quantum-network.sh")
+            dst="/tmp/quantum-network.sh",
+            juju_home=self.config.juju_home(True))
         utils.remote_run(unit.machine_id,
-                         cmds="sudo chmod +x /tmp/quantum-network.sh")
+                         cmds="sudo chmod +x /tmp/quantum-network.sh",
+                         juju_home=self.config.juju_home(True))
         utils.remote_run(unit.machine_id,
-                         cmds="sudo /tmp/quantum-network.sh")
+                         cmds="sudo /tmp/quantum-network.sh",
+                         juju_home=self.config.juju_home(True))
```

Comment:
a style nitpick here, bare "True" args in a function are not very readable.
IMO a better alternative in this case would be to change these to include the keyword arg's name, e.g. `juju_home=self.config.juju_home(use_expansion=True)`